zulip/integrations/zephyr/check-mirroring

```diff
@@ -43,11 +37,11 @@ sys.path[:0] = [os.path.join(options.root_path, "api/"),
 mit_user = 'tabbott/extra@ATHENA.MIT.EDU'
 
 sys.path.append(".")
-import zulip
```

Comment:
If you read the comment above, you'd have known that this was here for a reason.  We want to adjust `sys.path`  before trying to import `zulip`.  What I think you could do is parse the args twice; first, above, just parse `root_path`, and then below here, reparse with the full set of arguments using `zulip_default_args`.

Reply:
@timabbott: In all honesty, I am a bit confused. If you look at the code following the comment that you just mentioned, I think that the adjustment made to `sys.path` is redundant seeing that the "zulip" is a package now and not a directory under `<options.root_path>/api/` that needs to be added to `sys.path` before making the import. Even if you do add it to `sys.path` the, fact is that you would still be importing the `zulip` package (and not some local directory) because the `api/zulip/` doesn't even exist anymore. So isn't importing `zulip` independent of the `sys.path` manipulation in this case now that zulip is an installed package and not necessarily a module just lying in the root of one's `zulip` directory?

I think we talked about how changing the whole redundant `sys.path` modifications would be a follow-up to this PR, but that was few weeks ago, I think. Again, my apologies for not catching this and explaining it earlier but I'd love to know your thoughts about this! :)

Reply:
This script is deployed independently from the rest of the package in one of its 2 main places it's used, for historical reasons.  I suppose I could migrate it to not do that so we can eliminate this feature, but that's what's going on.  Let me see if I find time to investigate doing so today.